SPECS/autoconf/autoconf.spec

```diff
@@ -1,14 +1,13 @@
 Summary:	    The package automatically configure source code
 Name:		    autoconf
-Version:	    2.69
-Release:        11%{?dist}
+Version:	    2.71
+Release:        1%{?dist}
 License:	    GPLv2
-URL:		    http://www.gnu.org/software/autoconf
+URL:		    https://www.gnu.org/software/autoconf/
```

Comment:
Small request: could run a linter against the .spec files to get rid of tabs/fix alignment?

Reply:
Sure thing